R/format_ae_specific.R

```diff
@@ -214,21 +230,96 @@ format_ae_specific <- function(outdata,
     within_tbl
   }
 
+  res <- data.frame(name = outdata$name, res)
+
+  # Additional options for AE specific
+  if ("prepare_ae_specific" %in% as.character(outdata$prepare_call)) {
+    if ("soc_name" %in% names(outdata)) {
+      soc_name <- outdata$soc_name
+    }
+
+    # Filtering by criteria
+    if (filter_criteria > 0) {
+      if (toupper(filter_method) == "PERCENT") {
```

Comment:
If we have `match.arg()`, I guess we no longer need the `toupper()`? If you agree, then there is a similar issue in line 261, 280, 289.